kernel/src/fs/ramfs/fs.rs

```diff
@@ -1189,6 +1189,17 @@ impl Inode for RamInode {
     fn extension(&self) -> Option<&Extension> {
         Some(&self.extension)
     }
+
+    fn check_permission(&self, req: crate::fs::utils::RequestPermission) -> Result<()> {
```

Comment:
Is the implementation consistent with Linux's ramfs/tmpfs, where only the owner's permissions need to be checked?

Reply:
I think another reason might be that the case in `rootfs::init()` differs from the current default implementation of `check_permission()`. At this early stage, the multi-user module isn't fully developed. Perhaps we could modify `check_permission()` by adding another parameter to accommodate these situations, although I'm not certain. My instinct is that once the kernel is fully initialized, ramfs should have the same permission checks as other file systems. What do you think?

Reply:
Agree with your point. In this PR, I will add a parameter to check_permission() as a temporary fix.

Reply:
The issue has been addressed. File permission checks are now bypassed during the bootstrap context.